### ISSUE TITLE
new plugin signature

### ISSUE THREAD

**Issue by fdncred** [JosephTLyons/nu_plugin_periodic_table#17]
Head's up. Nushell 0.75.1 just landed some enhancements for plugins. This will be in nushell 0.76.0.

1. We need to use `PluginSignature` vs `Signature` and use `nu_protocol::PluginSignature`
2. We need to use `PluginExample` if you want examples in your plugin, from `nu_protocol::PluginExample`

This is an example of how I've used this in a plugin I'm working on called nu_plugin_dbg
```rust
    fn signature(&self) -> Vec<PluginSignature> {
        vec![PluginSignature::build("dbg")
            .usage("View dbg results")
            .category(Category::Experimental)
            .plugin_examples(vec![PluginExample {
                description: "View dbg results in a pipeline".into(),
                example: "ls | dbg | get name | dbg".into(),
                result: None,
            }])]
    }
```

**Comment by JosephTLyons** [comment]
Thanks again for the update @fdncred - much appreciated.  I'll get these changes in.

**Comment by JosephTLyons** [comment]
Fixed in v0.2.5

https://github.com/JosephTLyons/nu_plugin_periodic_table/commit/6ddec4ade0472dc9f5a7a5586936808ff6a21ce1

Thanks again for the heads up @fdncred.
